addons/web_editor/static/src/scss/wysiwyg_iframe.scss

```diff
@@ -15,7 +15,6 @@ iframe.wysiwyg_iframe.o_fullscreen {
 
 body.o_in_iframe {
     background-color: $o-view-background-color;
-    display: flex;
```

Comment:
Seems a bit too global, are we sure it's ok to remove that for all usages?

Reply:
I adapted the code where I needed to and didn't find any place where that seemed to cause issues.